src/App.cpp

```diff
@@ -1,15 +1,74 @@
 #include "App.h"
 
-App::App(int width, int height, std::string appName) : width(width), height(height), appName(appName) 
+// temp
+void message_callback(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length, GLchar const *message, void const *user_param)
```

Comment:
To jest nadal potrzebne?

Reply:
usunąłem, może dodam potem pod jakimś skrótem klawiszowym włączanie debugowania